File: app/src/main/java/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/VeilShatterer.java
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances;

import it.paranoidsquirrels.idleguildmaster.R;
import it.paranoidsquirrels.idleguildmaster.storage.data.items.Item;

/* JADX INFO: loaded from: classes3.dex */
public class VeilShatterer extends Item {
    @Override // it.paranoidsquirrels.idleguildmaster.storage.data.items.Item
    protected void configureProperties() {
        this.idName = R.string.item_veil_shatterer_name;
        this.idDescription = R.string.item_veil_shatterer_description;
        this.idImage = R.drawable.veil_shatterer;
        this.price = 90L;
    }
}

File: app/src/main/kotlin/it/paranoidsquirrels/idleguildmaster/storage/data/items/instances/VeilShatterer.kt
package it.paranoidsquirrels.idleguildmaster.storage.data.items.instances

import it.paranoidsquirrels.idleguildmaster.R
import it.paranoidsquirrels.idleguildmaster.storage.data.items.Item

class VeilShatterer : Item() {
    override fun configureProperties() {
        idName = R.string.item_veil_shatterer_name
        idDescription = R.string.item_veil_shatterer_description
        idImage = R.drawable.veil_shatterer
        price = 90L
    }
}
